cmd/earthly/root_cmds.go

```diff
@@ -35,15 +35,16 @@ func (app *earthlyApp) rootCmds() []*cli.Command {
 		{
 			Name:        "build",
 			Usage:       "Build an Earthly target",
-			Description: "Build an Earthly target",
+			Description: "Build an Earthly target.",
```

Comment:
Updated all usage strings to not have `.` for consistency and added them to descriptions also just for consistency across the help outputs for all commands to match.

Reply:
Updated with tests for this for each of the files that have Usages and Descriptions